### PR TITLE
Adding performance test scripts and test artifacts

### PR DESCRIPTION
**Pull request by dilee**
## Purpose
Adding performance test scripts and test artifacts for EI

### CODE REVIEW

**Review by chrishantha — APPROVED** [review]

File: components/perftest-capp/src/main/capp/CBRProxy_1.0.0/CBRProxy-1.0.0.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy name="CBRProxy" startOnLoad="true" transports="https http" xmlns="http://ws.apache.org/ns/synapse">
```
Comment (prabushi) [inline components/perftest-capp/src/main/capp/CBRProxy_1.0.0/CBRProxy-1.0.0.xml:2]: Add licence header for synapse configurations.

File: distribution/scripts/jmeter/requests/102400B_buyStocks.xml [hunk 2]
```diff
@@ -0,0 +1,996 @@
+<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
```
Comment (prabushi) [inline distribution/scripts/jmeter/requests/102400B_buyStocks.xml:1]: Format the xml payloads.

Comment (dilee) [reply distribution/scripts/jmeter/requests/102400B_buyStocks.xml:1]: Fixed in https://github.com/wso2/performance-ei/pull/9

File: components/perftest-capp/src/main/capp/CBRProxy_1.0.0/CBRProxy-1.0.0.xml (hunk 1 above)
Comment (dilee) [reply components/perftest-capp/src/main/capp/CBRProxy_1.0.0/CBRProxy-1.0.0.xml:2]: Fixed in https://github.com/wso2/performance-ei/pull/9

**Pull request merged**